openlibrary/plugins/books/readlinks.py

```diff
@@ -102,25 +101,17 @@ def get_solr_edition_records(iaids):
 class ReadProcessor:
     def __init__(self, options):
         self.options = options
-        self.set_inlibrary = False
-
-    def get_inlibrary(self):
-        if not self.set_inlibrary:
-            self.set_inlibrary = True
-            self.inlibrary = inlibrary.get_library()
-        return self.inlibrary
-
 
     def get_item_status(self, ekey, iaid, collections, subjects):
```

Comment:
@mekarpeles Is this `readlinks.get_item_status()` method use anywhere in the current lending system? It looks like it should be `@deprecated` or removed.  I believe using subjects to determine lending status is fully deprecated, and the subjects are being cleared (see #2107) -- if this _is_ still used, I don't think it will be giving good results.